sanitize.py

```diff
@@ -122,10 +122,26 @@ def check_sanity(ipynbFile: str = '') -> bool:
             print("Failed outputs check for cell")
             rc = False
 
-        skip_pattern = "task.configure(credentialsJson="
-        if skip_pattern in cell.get('source'):
-            print("Failed credentialsJson/code check for cell")
-            rc = False
+        # Look for this pattern
+        # "task.configure(credentialsJson='''{\n",
+        # "    \"credential_name\": abc,
+        # "    \"credential_type\": def",
+        # "    \"credential_id\": ghi",
+        # "}''')\n",
+
+        # This pattern is ok
+        # "task.configure(credentialsJson='''{\"credential_type\": \"" + md.action_type + "\",}''')"
+
+        if cell.get('metadata').get('legotype') is None:
+            continue
+
+        action_type = cell.get('metadata').get('legotype').replace("LEGO", "CONNECTOR")
+        skip_pattern = 'task.configure(credentialsJson='
+        ok_pattern = "task.configure(credentialsJson='''{\\\"credential_type\\\": \\\"" + action_type + "\\\",}''')"
```

Comment:
We need to fix the sanitize part as well which the devs run to fix the runbook.

Reply:
Done